Earthfile

```diff
@@ -467,23 +467,39 @@ base-image:
 
     RUN rm -rf /tmp/*
 
-    DO +CONTAINER_IMAGE_VERSION -VERSION=${OS_VERSION}
-    ARG _CIMG=$(cat IMAGE)
+    COPY ./naming.sh .
+    RUN ./naming.sh container_artifact_name
+    RUN ./naming.sh container_artifact_name > ./IMAGE
 
     # luet cleanup
     RUN luet cleanup
     RUN rm -rf /var/luet
 
+    ARG _CIMG=$(cat ./IMAGE)
```

Comment:
are we using this IMAGE file anywhere else? Should we just make this `_CIMG=$(/naming.sh container_artifact_name)` ?

Reply:
We do in some workflows:

```
~/workspace/kairos/kairos (1898-byoi)*$ git grep "cat IMAGE"
.github/workflows/reusable-build-flavor.yaml:          IMAGE=$(cat IMAGE)
.github/workflows/reusable-build-flavor.yaml:          _IMG=$(cat IMAGE)
.github/workflows/reusable-build-provider.yaml:          _IMG=$(cat IMAGE)
```

Ideally we shouldn't include it in the images we produce but it was like that before this PR. We can try to fix it outside it.